src/test/java/lotto/domain/LottoNumberTest.java

```diff
@@ -0,0 +1,33 @@
+package lotto.domain;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+
+public class LottoNumberTest {
+
+    @ParameterizedTest
+    @ValueSource(ints = {1, 44})
```

Comment:
로또의 범위 값에 대한 테스트를 하시려 한 것 같아요!
이미 알고 계시겠지만 로또는 1~45의 값이죠! 😄 
```suggestion
    @ValueSource(ints = {1, 45})
```